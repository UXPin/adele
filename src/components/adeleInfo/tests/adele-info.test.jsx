import { mount } from 'enzyme';
import React from 'react';
import 'jest-styled-components';

import AdeleInfo from '../adele-info';

it('renders AdeleInfo correctly', () => {
  const wrapper = mount(<AdeleInfo />);
  expect(wrapper).toMatchSnapshot();
});
