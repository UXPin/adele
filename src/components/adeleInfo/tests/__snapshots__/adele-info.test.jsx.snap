// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AdeleInfo correctly 1`] = `
.c0 {
  width: 600px;
  margin: 0 auto;
}

.c1 {
  font-family: proxima-nova,Proxima Nova,Helvetica Neue,Helvetica,Arial,sans-serif;
  font-size: 42px;
  color: #1e1e1e;
  font-weight: 300;
  text-align: center;
  margin: 0;
}

.c2 {
  display: block;
  font-family: proxima-nova,Proxima Nova,Helvetica Neue,Helvetica,Arial,sans-serif;
  font-size: 16px;
  color: #777777;
  font-weight: 300;
  line-height: calc(16px * 2);
  margin: 40px auto 0 auto;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c4 {
  position: relative;
  outline: none;
  border: none;
  background: none;
  font-family: proxima-nova,Proxima Nova,Helvetica Neue,Helvetica,Arial,sans-serif;
  font-size: 16px;
  color: #777777;
  font-weight: 300;
  cursor: pointer;
}

.c4:hover {
  text-decoration: underline;
  -webkit-transition: all ease-in 0.2s;
  transition: all ease-in 0.2s;
}

.c4:focus {
  color: #006cff;
}

<AdeleInfo>
  <styled.section>
    <section
      className="c0"
    >
      <styled.h2>
        <h2
          className="c1"
        >
          Adele Goldberg. The Icon.
        </h2>
      </styled.h2>
      <styled.span>
        <span
          className="c2"
        >
          <p>
            No. Adele – the Design Systems Repository, is not named after Adele – the singer. This is a tribute to one of the most important computer scientists focused on graphic user interfaces, design patterns and object-oriented programming –
             
            <a
              href="https://en.wikipedia.org/wiki/Adele_Goldberg_(computer_scientist)"
              rel="noopener noreferrer"
              tabIndex={5}
              target="_blank"
            >
              Adele Goldberg
            </a>
            .
          </p>
          <p>
            Adele Goldberg worked at XEROX PARC in the 70s and managed the System Concepts Laboratory where, together with Alan Kay and others, she developed Smalltalk-80 – an object-oriented, dynamically typed programming language that was meant to power the "human-computer symbiosis".
          </p>
          <p>
            Needless to say, SmallTalk also pioneered many concepts important to all modern design systems. Objects in Smalltalk were easily transferable between applications and customizable. Smalltalk also served as the foundation of PARC's work on graphically based user interfaces (many GUI concepts have been developed by Adele Goldberg and her group!).
          </p>
          <styled.footer>
            <footer
              className="c3"
            >
              <p>
                Thank you Adele!
              </p>
              <NavLink
                action={[Function]}
                label="Back to the top"
                tab={5}
              >
                <styled.button
                  onClick={[Function]}
                  tabIndex={5}
                >
                  <button
                    className="c4"
                    onClick={[Function]}
                    tabIndex={5}
                  >
                    Back to the top
                  </button>
                </styled.button>
              </NavLink>
            </footer>
          </styled.footer>
        </span>
      </styled.span>
    </section>
  </styled.section>
</AdeleInfo>
`;
